src/scf-release/src/acceptance-tests-brain/test-resources/nfs_secgroup.json

```diff
@@ -1,11 +1,11 @@
 [
     {
-        "destination": "192.168.77.77",
+        "destination": "192.168.47.109",
```

Comment:
This is just a placeholder, the actual value does not matter.
The test overwrites it with the actual IP (because it has to work under Jenkins, etc. as well)

`src/scf-release/src/acceptance-tests-brain/test-scripts/011_nfspersi_test.rb`:79:

```
# Fix IP in the various configuration files
SGROUP = "#{tmpdir}/nfs_secgroup.json"
nfs_secgroups = File.open(resource_path('nfs_secgroup.json')) { |f| JSON.load(f) }
nfs_secgroups.map! { |sg| sg.tap { |sg| sg['destination'] = SADDR } }
File.open(SGROUP, 'w') { |f| f.puts nfs_secgroups.to_json }
```

Reply:
Ah, thanks for pointing that out.